packages/models/proto/v2/authorization/client.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package authorization.v2;
+
+import "v2/authorization/key.proto";
+
+message ClientV2 {
+  string id = 1;
+  string consumerId = 2;
+  string name = 3;
+  optional string description = 4;
+  repeated string relationships = 5;
+  repeated string purposes = 6;
+  ClientKindV2 kind = 7;
+  optional int64 createdAt = 8; // to do double-check why optional
+  repeated string users = 9;
+  repeated KeyV2 keys = 10;
```

Comment:
@galales  Should this be `KeyV2` or `KeyEntryV2`? If not needed, I could delete [this](https://github.com/pagopa/interop-be-monorepo/pull/529/files#diff-e867ec36aa770f378c540afdb1ed7a26872da1d507fb51b7c861d95858899ebcR5-R8).

Reply:
More info: if `ClientKeyAddedV2` and `ClientKeyDeletedV2` embed `kid`, there is no need to add an extra `keyId` in the parent structure.

Reply:
yes this should be `KeyV2` and we can remove `KeyEntryV2`.

Not sure what you mean with the second comment. The `kid` in the event is the identifier of the targeted object inside the main `client` object, like the `descriptorId` in a `EserviceDescriptorAdded` event.
But I may have misunderstood your point

Reply:
Sorry for the second comment. I might have got confused a bit.